airbyte-integrations/connectors/source-postgres/src/main/resources/internal_models/internal_models.yaml

```diff
@@ -10,13 +10,24 @@ properties:
     "$ref": "#/definitions/XminStatus"
   ctid_state:
     "$ref": "#/definitions/CtidStatus"
+  standard_state:
+    "$ref": "#/definitions/StandardStatus"
 definitions:
   StateType:
     description: Enum to define the sync mode of state.
     type: string
     enum:
+      - standard
       - xmin
       - ctid
+  StandardStatus:
+    type: object
+    extends:
+      type: object
+      existingJavaType: "io.airbyte.integrations.source.relationaldb.models.DbStreamState"
```

Comment:
the existing type has no state_type and version.
Is it possible to extend it to add these two fields?
if not can we refine it?

Reply:
That's what I did. I extended the DbStreamState and added a state_type key on L29-L30 below

Reply:
I see the `extends`.
Can you add a version also?